ntbs-service/Pages/Notifications/INotificationLayoutWithBanner.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ntbs_service.Models;
+
+namespace ntbs_service.Pages.Notifications
+{
+    public interface INotificationLayoutWithBanner
```

Comment:
Just a small naming preference - this interface is not actually a layout (nor the interface of a layout), so perhaps `IWithNotificationBanner` will suffice?